compose.yml

```diff
@@ -21,26 +21,26 @@ services:
     ports:
       - 9944:9944
       - 30333:30333
-    volumes: 
+    volumes:
       - data-alice:/data
-  
+
   node-bob:
-    build: 
+    build:
       context: .
       dockerfile: local.Dockerfile
     image: ${NODE_IMAGE:-mangatasolutions/rollup-node:local}
-    command: 
-      - --bob
+    command:
+      - --baltathar
       - --chain=rollup-local
       - --base-path=/data
       - --rpc-cors=all
       - --unsafe-rpc-external
       - --bootnodes=/dns/node-alice/tcp/30333/p2p/12D3KooWEyoppNCUx8Yx66oV9fJnriXwCcXwDDUA2kj6vnc6iDEp
     ports:
       - 9946:9944
-    volumes: 
+    volumes:
       - data-bob:/data
 
 volumes:
   data-alice:
```

Comment:
Don't you also wanna change the volume names?

Reply:
done!